metadata.json

```diff
@@ -114,6 +114,12 @@
         "10.15",
         "11"
        ]
+    },
+    {
+      "operatingsystem": "Rocky",
+      "operatingsystemrelease": [
+        "8"
+       ]
```

Comment:
Ideally, `metadata.json` contains platforms that are officially supported (and tested internally) by Puppet. Right now we do not test this module against Rocky Linux so we cannot validate that everything works properly.

Until that happens we should not add the platform to this file.

Reply:
I have just added the tests for the Rocky, is this enough?

Reply:
Thanks for adding tests! However this only validates the task behavior, not the module itself (stuff from the manifests/ directory), so until we get that done we cannot update `metadata.json`.

Module upgrade tests run on our internal infrastructure, and we'll be able to officially support Rocky when we start testing the module internally.

It would be good if you can specify in the commit message that this adds just _task_ support, as there may be other work needed for the module itself.

Reply:
I just corrected the commit description (d02ba8b30ba88335b5cbaf970a6eff67dd341cc1).
Should I delete the addition in the metadata.json file?

Reply:
Yes, please. We will add it back once we start testing the platform in our internal CI.

Reply:
Done! It's ok for you ?